app/routes/articles/ArticleDetailRoute.ts

```diff
@@ -104,8 +106,8 @@ export default compose(
         content: props.article.description,
       },
       {
-        property: 'article:author',
-        content: `${config.webUrl}/users/${props.author.username}`,
+        property: 'article:authors',
+        content: `${config.webUrl}/users/${props.authors.username}`,
```

Comment:
Ref. that authors is now an array, this will fail.

See https://ogp.me/#type_article for the correct way to handle multiple authors in the meta tags here